Assets/Scripts/InteractionDetect.cs

```diff
@@ -101,5 +124,37 @@ void Update()
                 }
             }
         }
+        // else

```

Comment:
I assume that all this commented out code is just in flux, so I am not reviewing it.